examples/metrics/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "example-metrics",
+  "private": true,
+  "version": "0.4.0",
+  "description": "Example of using @opentelemetry/metrics",
+  "main": "index.js",
+  "scripts": {
+    "start:observer": "node metrics/observer.js"
```

Comment:
Some reason this is `start:observer` instead of just `start`?

Reply:
I have created metrics example to be a place for other metrics too, so next step when working on documentation and examples should be to add `start:counter`, `start:measure`